schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/transform/JasContactMessageTransformer.java

```diff
@@ -0,0 +1,38 @@
+package com.jasify.schedule.appengine.spi.transform;
+
+import com.google.api.server.spi.config.Transformer;
+import com.jasify.schedule.appengine.model.message.ContactMessage;
+import com.jasify.schedule.appengine.spi.dm.JasContactMessage;
+import com.jasify.schedule.appengine.util.KeyUtil;
+
+/**
+ * @author wszarmach
+ * @since 07/11/15.
+ */
+public class JasContactMessageTransformer implements Transformer<ContactMessage, JasContactMessage> {
```

Comment:
Like I said the other day when we chatted... It used to be that the Transformer was a simple BeanUtils operation.  Now i seems to me like the concept of going from `@Model` to `JasXXX` is just pure overhead.